ui/cypress/integration/error-pages.spec.js

```diff
@@ -0,0 +1,75 @@
+describe('Error Pages Navigation', () => {
+  beforeEach(() => {
+    cy.setupMocks();
+    cy.login();
+  });
+
+  it('brings me to 404 error page', () => {
+    cy.visit('/404');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page404"]').should('have.length', 1);
+  });
+
+  it('redirects me to 404 error page if route not found', () => {
+    cy.visit('/undefined-route-that-doesnt-exists');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page404"]').should('have.length', 1);
```

Comment:
@ChengYanJin do you mean testing if the 404 text is rendered properly in there ?

I think that may be a bad idea because this text is directly into core-ui ErrorPage component, meaning if we update the text in core-ui it would make the metal tests fail.

Reply:
I think we don't need to necessarily to the actual text, but add an `aria-label` for the text element in core-ui